package.json

```diff
@@ -59,41 +59,41 @@
     "classnames": "^2.2.5",
     "events": "^1.1.0",
     "hogan.js": "^3.0.2",
-    "preact": "^10.0.0",
+    "preact": "^10.3.4",
```

Comment:
We shouldn't bump the `dependencies` because they restrict the available range for our users unnecessarily.

Reply:
Ups, nice catch. Indeed!